docker/docker-compose.yml

```diff
@@ -93,6 +93,30 @@ services:
       timeout: 2s
       retries: 20
 
+  # Vitess (~Planetscale)
+  # https://vitess.io/docs/18.0/get-started/vttestserver-docker-image/
+  # mysql -h localhost -P 33807 --protocol=TCP -u root -p
+  vitess-fk:
+    image: janprisma/vttestserver:mysql80@sha256:bc3978c2ec58f0ded93280f79f00d99741a60e728a14989b473b75fb6d8282b8
```

Comment:
We probably want to go to the latest official one, that was an attempt to do something with my own fork that did not actually work out in the end :D